requirements/base.in

```diff
@@ -1,6 +1,5 @@
 # Core requirements for using this application
 
-Django>=1.8,<2.0          # Web application framework
```

Comment:
We should probably keep Django listed here, just don't impose a maximum version.

Reply:
done